data/net/minecraft/util/Mth.mapping

```diff
@@ -238,12 +292,47 @@ CLASS net/minecraft/util/Mth
 			COMMENT Start value for the lerp
 		ARG 2 end
 			COMMENT End value for the lerp
+	METHOD lerp2 (DDDDDD)D
+		ARG 0 delta
+		ARG 2 delta2
+		ARG 4 start
+		ARG 6 end
+		ARG 8 start2
+		ARG 10 end2
+	METHOD lerp3 (DDDDDDDDDDD)D
+		ARG 0 delta
+		ARG 2 delta2
+		ARG 4 delta3
+		ARG 6 start
+		ARG 8 end
+		ARG 10 start2
+		ARG 12 end2
+		ARG 14 start3
+		ARG 16 end3
+		ARG 18 start4
+		ARG 20 end4
 	METHOD lfloor (D)J
 		COMMENT Long version of floor()
 		ARG 0 value
 	METHOD log2 (I)I
 		COMMENT Efficiently calculates the floor of the base-2 log of an integer value.  This is effectively the index of the highest bit that is set.  For example, if the number in binary is 0...100101, this will return 5.
 		ARG 0 value
+	METHOD map (DDDDD)D
+		ARG 0 input
+		ARG 2 inputMin
+		ARG 4 inputMax
+		ARG 6 outputMin
+		ARG 8 outputMax
+	METHOD map (FFFFF)F
+		ARG 0 input
+		ARG 1 inputMin
+		ARG 2 inputMax
+		ARG 3 outputMin
+		ARG 4 outputMax
```

Comment:
This doesn't make as much sense to me. I understand what you're going for as the first three take in the from and to values to get the delta and then maps it to the output. I don't know what would be clearer though without a javadoc.

Reply:
Not sure what to do here.